data/FilterList.json

```diff
@@ -14834,5 +14834,57 @@
     "syntaxId": 1,
     "viewUrl": "https://raw.githubusercontent.com/DandelionSprout/adfilt/master/Adblock%20list%20templates/Adblock%20list%20template%20-Hosts%20and%20uBlock%20Origin-.txt",
     "viewUrlMirror1": "https://repo.or.cz/FilterMirrorRepo.git/blob_plain/refs/heads/master:/Adblock%20list%20templates/Adblock%20list%20template%20-Hosts%20and%20uBlock%20Origin-.txt"
+  },
+  {
+    "id": 1294,
```

Comment:
and, @bogachenko submitted a few PRs stealing the next few IDs, so we'll need to increment some of these IDs to avoid conflicts as well. :)

Thanks.

Reply:
:dancer:

Reply:
By the way @collinbarrett, I wanted to ask for a long time, what is the period of updating the site? ie how quickly fixes (commits) will appear on your site?

Reply:
It should be updated in 24 hours or less. I do want to bring that number down in the near future with some work I'm doing on the DevOps side, but for now, there is some caching involved to reduce load.

Reply:
@collinbarrett okey. thanks